### ISSUE TITLE
Implement default font so drawer.text can be called directly

### ISSUE THREAD

**Issue by hamoid** [openrndr/openrndr#123]
**Is your feature request related to a problem?**
If the user calls `drawer.text()` nothing is visible unless a font is specified earlier

**Describe the solution you'd like**
Maybe `text()` detects if the font is undefined. In that case it loads a default font. Another option is to load a default font when starting the program, but that would add some start up latency and it's not needed if `.text()` is never used.

I can try implement it if an approach is suggested.

**Comment by edwinRNDR** [comment]
I think attempting to load `data/fonts/default.otf` at 16.0 is a good start. Then we rename the font in openrndr-template to `default.otf`. The idea here is to not embed fonts in OPENRNDR.

**Comment by hamoid** [comment]
At which point in the life cycle would it attempt to load the font?

**Comment by edwinRNDR** [comment]
That would be inside Drawer.text() and Writer.text()

**Comment by hamoid** [comment]
This seems to almost work:

https://github.com/openrndr/openrndr/compare/master...hamoid:defaultFont

In this example:

```
fun main() = application {
    program {
        extend {
            drawer.translate(50.0, 50.0)
            writer {
                newLine()
                text("A")
                newLine()
                text("B")
                newLine()
                text("C")
            }
            drawer.text("Hello", 250.0, 50.0)
        }
    }
}
```
The letter A is not shown. So the first call to `writer.text()` does not work but next ones do. 

If I move `drawer.text()` above `writer.text()` it does work right. 

I tried calling `flush` immediately after loading the font but it didn't work. It seems I need to initialize something after loading the font, but what?

**Comment by hamoid** [comment]
Implemented like this:
https://github.com/openrndr/openrndr/pull/137

**Comment by hamoid** [comment]
I accidentally just wrote this comment in the related pull request:

> The default-font feature was working with OPENRNDR 0.3 but no longer does with 0.4. Needs to be ported / updated.

I did that because GitHub changed the behavior of the search box and I didn't notice: before the search for issues and for pull requests were separate. Now when you search you find both in the same result list. I guess it's a good thing, but I didn't expect this. Need to observe the icons on the left side of the results.

**Comment by hamoid** [comment]
- [ ] Fix default font in OPENRNDR 0.4
